.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: "daily"
```

Comment:
```suggestion
    interval: "weekly"
```

Reply:
Thanks for the thorough response!

> Just listing out the tradeoffs we're making here as we evaluate this decision. However, since this is an easily reversible decision, regardless of which way we go, I would suggest we make a quick decision and iterate - we can always change the config later.

Sure, I'm all for iterating quickly. Ideally things like this start off as a discussion in an issue rather than a drive-by PR which gives us (maintainers) a chance to ask questions and discuss whether this makes sense for the project before moving to the PR stage. 

> Daily/Weekly schedule interval

I agree with most of the points you're making here but think this fails to recognize an important point: time. Like I said before, it gets noisy and it's hard to keep up with. 

You're coming in here proposing this change, which has value, but are you willing to be the one to maintain it? As you can see from the backlog of issues, we have a lot of other work to be done and merging this comes with a high cost.

> We will be informed about new versions more quickly, which can be important for security releases

I like the sound of that, but do we have evidence to say this is something we need? Probably not?

> Dependabot's PR Workflow vs Console

Thanks for explaining the differences! Sounds like this approach is preferable.

Okay just going to summarize my final thoughts here:
- next time, will you open up an issue so we discuss this before you open a PR?
- we can approve this and merge if you're willing to take ownership on reviewing and merging these PRs

Reply:
Sure, I'd be happy to write an issue next time. I didn't bother doing that because I didn't think this change would be objectionable :smile: 

> I like the sound of that, but do we have evidence to say this is something we need? Probably not?

As software maintainers, we're responsible for the supply chain and making sure that we have a clean bill of health with respect to both our code, as well as third-party dependencies that we depend on.

Here's some evidence, the outstanding advisories shown by `yarn audit` run against the latest `main` branch; it's possible that these aren't real issues, but it seems like good hygiene to stay up-to-date, provided that the risk to upgrade isn't significant (and you're in a better position to assess that than I am):

```
[coder@jawnsy-m code-server]$ yarn audit
yarn audit v1.22.10
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ ini                                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >1.3.6                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ stylelint                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ stylelint > global-modules > global-prefix > ini             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1589                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ moderate      │ Use of a Broken or Risky Cryptographic Algorithm             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ elliptic                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=6.5.4                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ parcel-bundler                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ parcel-bundler > node-libs-browser > crypto-browserify >     │
│               │ browserify-sign > elliptic                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1648                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ moderate      │ Use of a Broken or Risky Cryptographic Algorithm             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ elliptic                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=6.5.4                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ parcel-bundler                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ parcel-bundler > node-libs-browser > crypto-browserify >     │
│               │ create-ecdh > elliptic                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1648                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high          │ Prototype Pollution in node-forge                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ node-forge                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >= 0.10.0                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ parcel-bundler                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ parcel-bundler > node-forge                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1561                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
4 vulnerabilities found - Packages audited: 1216
Severity: 1 Low | 2 Moderate | 1 High
Done in 1.44s.
```